jax/lax/lax.py

```diff
@@ -432,10 +436,37 @@ def concatenate(operands, dimension):
 
 Precision = xla_client.PrecisionConfig.Precision
 Precision.__str__ = lambda precision: precision.name
+PrecisionType = Any
 
-def conv_general_dilated(lhs, rhs, window_strides, padding, lhs_dilation=None,
-                         rhs_dilation=None, dimension_numbers=None,
-                         feature_group_count=1, precision=None):
+# lhs_spec and out_spec are lists containing
+#   [batch dim, feature dim, spatial dims ...]
+# rhs_spec is a list containing:
+#   [out feature dim, in feature dim, spatial dims ...]
```

Comment:
This looks redundant with the docstring?

Reply:
I merely moved this, but you are right. Removed.